test/JDBC/jdbc_schedule

```diff
@@ -77,6 +77,12 @@ ignore#!#BABEL-CHECK-CONSTRAINT-before-14_6-vu-verify
 ignore#!#BABEL-CHECK-CONSTRAINT-before-14_6-vu-cleanup
 ignore#!#BABEL-3646-before-14_6-vu-prepare
 ignore#!#BABEL-3646-before-14_6-vu-verify
+ignore#!#bbf_view_def_upgrade-before-14_5-vu-prepare
+ignore#!#bbf_view_def_upgrade-before-14_5-vu-verify
+ignore#!#bbf_view_def_upgrade-before-14_7-vu-prepare
+ignore#!#bbf_view_def_upgrade-before-14_7-vu-verify
+ignore#!#bbf_view_def_upgrade-before-15_2-vu-prepare
+ignore#!#bbf_view_def_upgrade-before-15_2-vu-verify
```

Comment:
1. vu stands for "version upgrade", so you don't need "upgrade" in the file name.
2. It seems that `before-14_7` and `before_15_2` files are identical. Then let's use `before-14_7-or-15_2`.